lib/widgets/main_app_bar.dart

```diff
@@ -1,161 +1,150 @@
-                  'Board Name',
-                  style: TextStyle(color: Colors.white, fontSize: 20.0),
-                ),
-                // Lock
-                GestureDetector(
-                  onTap: () => print('Lock'), // insert new screen here
-                  child: Icon(
-                    Icons.lock,
-                    color: Colors.white,
-                    size: 25.0,
-                  ),
-                ),
-              ],
-            ),
-          ),
-        ],
-      ),
-    );
-  }
-}
+import 'package:flutter/material.dart';
+// import 'widgets.dart';
+import 'package:overlay_screen/overlay_screen.dart';
+
+class MainAppBar extends StatelessWidget {
+  final double scrollOffset;
+
+  const MainAppBar({
+    Key key,
+    this.scrollOffset = 0,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    OverlayScreen().saveScreens({
+      'display': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: <Widget>[
+            SizedBox(
+              height: 200,
+              width: double.infinity,
+              child: DecoratedBox(
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                ),
+                child: Column(
+                  children: <Widget>[
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.assignment,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board One'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.assignment,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Two'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.backup,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Three'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      'edit_home': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.center,

```

Comment:
Hmm... does this mean there an overlay in the middle of the screen? Not sure - just wondering!